app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -13,284 +13,224 @@
-                      />
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="au-o-grid">
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <AuLabel for="administrative-unit-scope">Werkingsgebied</AuLabel>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      {{!-- TODO Make this work --}}
-                      <PowerSelectMultiple
-                        @disabled={{true}}
-                        @allowClear={{true}}
-                        @searchEnabled={{true}}
-                        @loadingMessage="Aan het laden..."
-                        @noMatchesMessage="Geen resultaten"
-                        @searchMessage="Typ om te zoeken"
-                        @selected={{@model.administrativeUnit.scope}}
-                        @onChange={{fn (mut @model.administrativeUnit.scope)}}
-                        @triggerId="administrative-unit-scope"
-                        as |scope|
-                      >
-                        {{scope}}
-                      </PowerSelectMultiple>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div class="au-o-grid">
-                {{#each @model.administrativeUnit.identifiers as |identifier|}}
-                  <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                    <div class="au-o-grid">
-                      <div class="au-o-grid__item au-u-1-4">
-                        <p><strong>{{identifier.idName}}</strong></p>
-                      </div>
-                      <div class="au-o-grid__item au-u-3-4">
-                        <p>{{identifier.structuredIdentifier.localId}}</p>
-                      </div>
-                    </div>
-                  </div>
-                {{/each}}
-              </div>
-            </c.content>
-          </AuCard>
-        </div>
+                  <AuControlRadio
+                    @label="Nee"
+                    @name="is-cross-border"
+                    @value={{@model.administrativeUnit.crossBorder}}
+                    @identifier="cross-border-false"
+                    @onChange={{fn
+                      (mut @model.administrativeUnit.crossBorder)
+                      false
+                    }}
+                    checked={{not @model.administrativeUnit.crossBorder}}
+                  />
+                </:content>
+              </Item>
+            </Grid>
+            {{/if}}
+            <Grid class="au-u-margin-top-large" as |Item|>
+              <Item @labelFor="administrative-unit-scope">
+                <:label>Werkingsgebied</:label>
+                <:content>
+                  {{! TODO Make this work }}
```

Comment:
Unrelated to this PR, but what was the blocker again? Do we have a JIRA issue for this?

Reply:
I don't know what was the blocker, maybe create the component because it's going to be all possible locations?